setup.sh

```diff
@@ -1,7 +1,16 @@
 #!/bin/zsh
 set -e
-docker build -f setup.dockerfile -o node .
-pnpx snarkjs zkey export verificationkey node/stark/stark_verify_final.zkey verification_key.json
+if [ ! -f "/stark/stark_verify" ]; then
+    echo "stark_verify not found, please copy it to /stark/stark_verify"
```

Comment:
Might be good to add some instructions here on how to do this, or prompt the user if they would like to if they are missing

Reply:
I will add some and fix the language but this will probably be easier to document after a few more rounds of changes, i.e. we have a way to upgrade the prover from a script or install recipe.

Reply:
I cleaned it up slightly, It's still not super helpful.   Lets go through the process of setting up a version repo for the prover and then we will modify the script to install the prover in general.